wifiphisher/common/phishinghttp.py

```diff
@@ -15,59 +16,60 @@
 
 class DowngradeToHTTP(tornado.web.RequestHandler):
 
-    def get(self, url):
+    def get(self):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
 
-    def get(self, url):
+    def get(self):
+        client_request = self.request.path[1:]
+        render_file = "index.html"
         try:
-            if "/" in url:
-                self.redirect("/index")
-            self.render("index.html", **template.get_context())
+            if self.request.path == "/":
```

Comment:
wouldn't this be better?
<script src="https://gist.github.com/blackHatMonkey/5980a30d679faeeeb31da15548abdef2.js"></script>